spec/factories/messages.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: messages
+#
+#  id               :bigint           not null, primary key
+#  messageable_type :string           not null
+#  text             :text             not null
+#  created_at       :datetime         not null
+#  updated_at       :datetime         not null
+#  messageable_id   :integer          not null
+#  user_id          :integer          not null
+#
+FactoryBot.define do
+  factory :message do
+    association :user
+    text { Faker::Lorem.paragraph }
+    messageable_id { user_id }
+    messageable_type { 'user' }
```

Comment:
Ditto with messageable.

Reply:
I don't understand what ditto references to

Reply:
I originally referred to [this comment](https://github.com/iTechArt-RubyLab/social-network/pull/13#discussion_r791674169) with question about `messageable` purpose, but I guess you have your own ways of doing things :)
While we are still on a subject: you don't have to specify columns from polymorphic assoc. separately, it's should be enough to do
```suggestion
    messageable { create(:user) } # or whatever you want as messageable
```

Reply:
`NoMethodError: undefined method `messageable='` is raised

Reply:
That's because your polymorphic association is not defined on your model, lol
Your model is missing something like
```ruby
belongs_to :messageable, polymorphic: true
```